tasks/main.yml

```diff
@@ -92,16 +92,15 @@
     group: "{{ pulp_user }}"
     force: false
 
-- name: Generate random secret for Django
-  shell: cat /dev/urandom | tr -dc 'a-z0-9!@#$%^&*(\-_=+)' | head -c 50
-  check_mode: false
+- name: Generate random secret for server.yaml
+  shell: tr -dc 'a-z0-9!@#$%^&*(\-_=+)' < /dev/urandom | head -c 50
   changed_when: false
-  register: random_secret
+  check_mode: false
+  register: result
```

Comment:
Why the generic ```result``` instead of ```random_secret```?

Reply:
This variable will stick around for the entire duration of the play, meaning that this variable is going to be visible to every task that's executed after this one. I don't know what roles might be called, or how they work internally. This is especially true given the use of roles from Ansible Galaxy. By using a variable by this name, I avoid accidentally configuring one of those roles, and I signal that this variable is a temporary one for local use.